x-pack/plugins/ml/public/components/json_tooltip/tooltips.js

```diff
@@ -0,0 +1,315 @@
+      defaultMessage: 'Defines the fields and functions used for analysis.'
+    })
+  },
+  new_job_influencers: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobInfluencersTooltip', {
+      defaultMessage: 'Select which categorical fields have influence on the results. \
+      Who/what might you \"blame\" for an anomaly? Recommend 1-3 influencers.'
+    })
+  },
+  new_job_detector_description: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorDescriptionTooltip', {
+      defaultMessage: 'User-friendly text used for dashboards.'
+    })
+  },
+  new_job_detector_function: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorFunctionTooltip', {
+      defaultMessage: 'Analysis functions to be performed e.g. sum, count.'
+    })
+  },
+  new_job_detector_fieldname: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorFieldnameTooltip', {
+      defaultMessage: 'Required for functions: sum, mean, median, max, min, info_content, distinct_count.'
+    })
+  },
+  new_job_detector_fieldname_subset: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorFieldnameSubsetTooltip', {
+      defaultMessage: 'Required for functions: sum, mean, max, min, distinct_count.'
+    })
+  },
+  new_job_detector_byfieldname: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorByfieldnameTooltip', {
+      defaultMessage: 'Required for individual analysis where anomalies are detected compared to an entity\'s own past behavior.'
+    })
+  },
+  new_job_detector_overfieldname: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorOverfieldnameTooltip', {
+      defaultMessage: 'Required for population analysis where anomalies are detected compared to the behavior of the population.'
+    })
+  },
+  new_job_detector_partitionfieldname: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorPartitionfieldnameTooltip', {
+      defaultMessage: 'Allows segmentation of modeling into logical groups.'
+    })
+  },
+  new_job_detector_excludefrequent: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorExcludefrequentTooltip', {
+      defaultMessage: 'If true will automatically identify and exclude frequently occurring entities which \
+      may otherwise have dominated results.'
+    })
+  },
+  new_job_data_format: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDataFormatTooltip', {
+      defaultMessage: 'Describes the format of the input data: delimited, JSON, single line or Elasticsearch.'
+    })
+  },
+  new_job_time_field: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobTimeFieldTooltip', {
+      defaultMessage: 'Name of the field containing the timestamp.'
+    })
+  },
+  new_job_time_format: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobTimeFormatTooltip', {
+      defaultMessage: 'Format of the time field: epoch, epoch_ms or Java DateTimeFormatter string. Important to get right.'
```

Comment:
@peteharverson should we move something from this string to values?

Reply:
This can be left as it is, in a single String